File: app/src/main/java/com/ferranpons/issposition/WebViewActivity.java
package com.ferranpons.issposition;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.webkit.WebView;

public class WebViewActivity extends AppCompatActivity {

  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_webview);
    WebView webView = (WebView) findViewById(R.id.webView);
    webView.loadUrl("https://github.com/ferranponsscmspain/iss-position");
  }
}

File: app/src/main/java/com/ferranpons/issposition/WebViewActivity.kt
package com.ferranpons.issposition

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.webkit.WebView

class WebViewActivity : AppCompatActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_webview)
        val webView = findViewById(R.id.webView) as WebView
        webView.loadUrl("https://github.com/ferranponsscmspain/iss-position")
    }
}
